src/utils/optionsMatches.js

```diff
@@ -0,0 +1,42 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
+
+const extractValueFromInput = (property, input) => {
```

Comment:
> whereas this function locks us into a finite set of properties, making it a little more difficult to add a new one, and will cause problems if/when some rule needs a special case.

If mitigating this is more of a priority than avoiding code duplication (mainly within the rules using `ignoreAtRules`), then I agree we should simply pass the value in.

@m-allanson Thanks for making a start on this! Can you foresee any issues with either passing in just the value or consolidating the utils? If not, would you mind giving it a go?